src/main/kotlin/no/nav/familie/oppdrag/config/ApiExceptionHandler.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.familie.oppdrag.config
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.oppdrag.common.RessursUtils.illegalState
+import no.nav.familie.oppdrag.common.RessursUtils.unauthorized
+import no.nav.security.token.support.spring.validation.interceptor.JwtTokenUnauthorizedException
+import org.springframework.core.NestedExceptionUtils.getMostSpecificCause
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
+class ApiExceptionHandler {
+
+    @ExceptionHandler(JwtTokenUnauthorizedException::class)
+    fun handleJwtTokenUnauthorizedException(jwtTokenUnauthorizedException: JwtTokenUnauthorizedException): ResponseEntity<Ressurs<Nothing>> {
+        return unauthorized("Unauthorized")
+    }
+
+    @ExceptionHandler(Throwable::class)
+    fun handleThrowable(throwable: Throwable): ResponseEntity<Ressurs<Nothing>> {
+        val mostSpecificThrowable = getMostSpecificCause(throwable)
```

Comment:
Burde vi logge feilen på noe vis? secure-log med stacktrace og evt. vanlig log for å si hva slags exception som har slått til?

Reply:
(My bad, ser at det skjer i illegalstate...)